parking_permits/admin_resolvers.py

```diff
@@ -252,21 +252,11 @@ def resolve_zone_by_location(obj, info, location):
 @query.field("customer")
 @is_customer_service
 @convert_kwargs_to_snake_case
-@audit_logger.autolog(
-    AuditMsg(
-        "Admin retrieved customer details.",
-        operation=audit.Operation.READ,
-    ),
-    autotarget=audit.TARGET_RETURN,
-    add_kwarg=True,
-)
-def resolve_customer(obj, info, audit_msg: AuditMsg = None, **data):
+def resolve_customer(obj, info, **data):
     query_params = data.get("query")
     customer = None
 
     if national_id_number := query_params.get("national_id_number"):
-        # We're searching data from DVV now, so change the event type.
-        audit_msg.event_type = audit.EventType.DVV
```

Comment:
Probably the whole `EventType.DVV`-enum should be removed, because it seem not to be used elsewhere in the code and we do not need that anymore.